docs/general/set-up-adguard-vpn-on-your-router.md

```diff
@@ -30,7 +30,7 @@ This option is only available if you have an AdGuard VPN subscription and your r
 
 1. In your admin page, go to the router settings.
 1. Enable *VPN Client* and click *Add VPN server*.
-1. Select IPsec-client (it may be called IKE; not L2TP/IPsec).
+1. Select IPsec-client (it may for some router brands be called IKE; not L2TP/IPsec).
```

Comment:
btw, shouldn't it be IKEv2 here?

Reply:
as here https://adguard-vpn.com/kb/general/set-up-adguard-vpn-on-your-router/#how-to-set-up-adguard-vpn-on-your-router

Reply:
Most probably. I wasn't sure what to choose since the GitHub text file had said IKE previously.